frontend/packages/console-shared/src/selectors/cluster-version.ts

```diff
@@ -0,0 +1,4 @@
+import * as _ from 'lodash';
+import { ClusterVersionKind } from '@console/internal/module/k8s';
+
+export const getCVChannel = (cv: ClusterVersionKind): string => _.get(cv, 'spec.channel');
```

Comment:
We're splitting this logic across modules. We should put it in the existing cluster-settings.ts. Eventually that should move into console-shared.

Reply:
```suggestion
export const getClusterVersionChannel = (cv: ClusterVersionKind): string => cv && cv.spec ? cv.spec.channel : undefined;
```

Reply:
done, moved to `cluster-settings.ts` along other selectors there.